src/components/NavBar.jsx

```diff
@@ -1,33 +1,56 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
Can this be avoided now? Generally eslint exceptions are a last ditch escape hatch